src/Backoffice/Users/App/Integrations/pVerify/Requests/EligibilitySummary.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Users\App\Integrations\pVerify\Requests;
+
+use Lightit\Backoffice\Users\Domain\DataTransferObjects\pVerifyEligibilitySummaryRequestDTO;
+use Saloon\Http\Request;
+
+class EligibilitySummary extends Request
+{
+    public function __construct(
+        protected pVerifyEligibilitySummaryRequestDTO $pVerifyEligibilitySummary,
```

Comment:
Casing in the class

Reply:
should we use capital P? I understand the name of the company is written with an lowercase p for a reason, that's why I didn't capitalize it.

Reply:
but it is a convention, classes should always start with uppercase